api.ts

```diff
@@ -1356,7 +1359,7 @@ export function makeSyncTable<
     const {result, continuation} = await wrappedExecute(params, context);
```

Comment:
this errors out for me as well. if undefined is returned, the unpack will throw.

Reply:
ahh ok will add -- how did you hit that case in the `fake.ts` testing?

Reply:
i changed Students sync table to return undefined in the default branch. and then 

```
./node_modules/.bin/ts-node cli/coda.ts execute test/packs/fake.ts Students random-string
```

Reply:
ah cool that makes sense